apis/v1alpha2/types.go

```diff
@@ -100,7 +75,7 @@ type ProviderConfigList struct {
 // +kubebuilder:printcolumn:name="CONFIG-NAME",type="string",JSONPath=".providerConfigRef.name"
 // +kubebuilder:printcolumn:name="RESOURCE-KIND",type="string",JSONPath=".resourceRef.kind"
 // +kubebuilder:printcolumn:name="RESOURCE-NAME",type="string",JSONPath=".resourceRef.name"
-// +kubebuilder:resource:scope=Cluster,categories={crossplane,provider,alibaba}
+// +kubebuilder:resource:scope=Cluster
```

Comment:
Was there a reason to remove these categories?

Reply:
Nope, I just copied the latest template from provider-template. I create a PR for the provider-template which might help new providers not omit the field https://github.com/crossplane/provider-template/pull/27.